### PR TITLE
Add flyway migrations

### LINKED ISSUE #7: Add Flyway for the testing Database
Add the Flyway Core: https://mvnrepository.com/artifact/org.flywaydb/flyway-core/9.10.0
Add the Flyway Maven Plugin: https://flywaydb.org/documentation/usage/maven/
Add create SQL Files for the testing Entities under `src/main/resources/db/migration`

### PR DESCRIPTION
**Pull request by Vichukano**
## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I used the given Project Style Guide

### CODE REVIEW

**Comment by Vichukano** [comment]
Pull request for https://github.com/Mom0aut/DynamicRelations/issues/7

File: testing/pom.xml [hunk 1]
```diff
@@ -45,5 +45,20 @@
             <artifactId>spring-context</artifactId>
             <version>${spring-context.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.flywaydb</groupId>
+            <artifactId>flyway-core</artifactId>
+            <version>9.10.0</version>
```
Comment (Mom0aut) [inline testing/pom.xml:51]: Pls extract the version number into `properties` like above 😄

File: testing/pom.xml [hunk 2]
```diff
@@ -45,5 +45,20 @@
...
+        </dependency>
     </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.flywaydb</groupId>
+                <artifactId>flyway-maven-plugin</artifactId>
+                <version>9.8.1</version>
```
Comment (Mom0aut) [inline testing/pom.xml:60]: Pls extract the version number into properties like above 😄

File: testing/pom.xml (hunk 1 above)
Comment (Vichukano) [reply testing/pom.xml]: done

File: testing/pom.xml (hunk 2 above)
Comment (Vichukano) [reply testing/pom.xml]: done

**Pull request merged**